WordPress/Tests/Arrays/ArrayDeclarationSpacingUnitTest.inc.fixed

```diff
@@ -2,7 +2,53 @@
 
 $good = array( 'value1', 'value2' ); // Ok.
 
-$query_vars = array( 'post_type' => 'food' ); // Bad, no spaces after opening and before closing parenthesis.
+$query_vars = array( 'food' ); // Bad, no spaces after opening and before closing parenthesis.
```

Comment:
It is fine here, as this is the fixed version of the file. It was bad before, in the unfixed version.

And as to the reason for making this a non-associative array, that is because it could no longer be on a single line then, as associative arrays have to be multiline. So to check spacing, it had to be changed to a non-associative array now.

Yes, both of these initially confused me as well. 😄